pages/services/beta-dcos-monitoring/0.5.0-beta/operations/prometheus/alert-rules/index.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: layout.pug
+navigationTitle: Alert Rules
+title: Alert Rules
+menuWeight: 30
+excerpt: Alert Rules
+render: mustache
+model: ../../../data.yml
+---
+
+#include /services/include/beta-software-warning.tmpl
+
+# Automatically loading Prometheus alert rules
+
+The {{ model.techName }} service can be configured to automatically load Prometheus alert rules from a Git repository.
+
+## Save Prometheus alert rules
+
+You should save your Prometheus alert rules (YAML format) in a Git repository.
+Assume that the repository is `https://github.com/company/alert-rules`.
+
+You can put the alert rules in a subfolder in the repository.
+For instance, `https://github.com/company/alert-rules/production`.
+
+```json
+{
+  "prometheus": {
+    "alert_rules_repository": {
+      "config_repository": {
+        "url": "https://github.com/company/alert-rules",
+        "path": "/production"
+      }
+    }
+  }
+}
+```
+
+<p class="message--important"><strong>IMPORTANT: </strong>Make sure that each of the alert rule files has suffix `.yml` and is a direct child of the subfolder (i.e., not nested under another subfolder).</p>
+
+The Git repository can be either public or private.
+If the Git repository is private, you will need to configure the credentials to access the Git repository (see below).
```

Comment:
Rewrite line #41 with the following:
If the Git repository is private, you will need to configure the credentials to access the Git repository as described below.